tox.ini

```diff
@@ -12,15 +12,10 @@ allowlist_externals = yapf
 skip_install = true
 commands = yapf -dr . {posargs}
 
-[testenv:isort]
-allowlist_externals = isort
+[testenv:ruff]
```

Comment:
@hirosassa [MUST] line2 should:

```diff
-envlist = py{39,310,311,312},yapf,isort,flake8,mypy
+envlist = py{39,310,311,312},yapf,ruff,mypy
```

Reply:
fixed in [b73d8d1](https://github.com/m3dev/gokart/pull/345/commits/b73d8d13cf8ae4996807e6b1046ae47846cd6f1e)